contents/handbook/people/team-structure/app-small-teams.md

```diff
@@ -0,0 +1,41 @@
+---
+title: App Small Teams
+sidebar: Handbook
+showTitle: true
+hideAnchor: false
+---
+
+App Small Teams are particular types of Small Teams that own an area of the product.
+
+## What does owning an area of the product mean?
+
+The app small team is responsible for everything related to that area of the product including:
+
+1. Usage
+2. Quality
+3. Revenue
+
+## What actions should the app small teams be doing for their area?
+
+- Each quarter:
+  1. Create good quarterly OKRs
+  2. Create an ambitious vision for their area
+- During the quarter:
+  1. Maintain a prioritized roadmap to help them achieve their objectives
+  2. Speak to customers
+  3. Monitor relevant metrics including those covering Usage, Quality and Revenue
+  4. Triage and fix related bugs
+  5. Assist the support hero in answering related questions
+  6. Collaborate with marketing
```

Comment:
I would also add a cheeky point here: "Become power users of their app area and use PostHog in their product development process" 
I know that we are doing this already so some extent, but I would love to specify this as part of this ways of working proposal